### PR TITLE
Tokenomics updates

### PR DESCRIPTION
**Pull request by astronautthis**
Staging links:
https://makerdao-1.gitbook.io/endgame-docs-staging/tokenomics/mkr-tokenomics
https://makerdao-1.gitbook.io/endgame-docs-staging/concepts/collateral-breakdown
https://makerdao-1.gitbook.io/endgame-docs-staging/concepts/peg-stability-mechanisms
https://makerdao-1.gitbook.io/endgame-docs-staging/tokenomics/sagittarius-engine

### CODE REVIEW

File: tokenomics/sagittarius-engine.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+The incentives for locking up MKR in the Sagittarius Engine are:
+- Token farming
+- Dai generation through vaults with MKR as collateral
+
+Withdrawing MKR from the Sagittarius Engine burns part of the deposited MKR.
+
+{% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
+
+## Entry and Exit
+When users deposit MKR into the Sagittarius Engine, they must delegate the deposited MKR through the [Easy Governance Frontend (EGF)](../maker-core/easy-governance-frontend.md). Depositors must specify a delegate's strategy smart contract when making their deposit.
```
Comment (blimpa) [inline tokenomics/sagittarius-engine.md:15]: "through _an_ EGF"?

File: tokenomics/sagittarius-engine.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+When a user withdraws their MKR from the Sagittarius Engine, 15% of the deposited MKR is burned.
+
+## MKR Vaults
+The Sagittarius Engine allows users to use deposited MKR as collateral to generate Dai, similar to normal Maker Vaults. The parameters for these vaults are below:
+
+#### [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) 
+There is no hard Debt Ceiling for these MKR vaults. However, a [Debt Ceiling Instant Access Module](https://manual.makerdao.com/module-index/module-dciam) will be configured to ensure that the Debt Ceiling does not increase too fast. A Soft Debt Ceiling parameter is set based on the value of all protocol-owned and SubDAO-owned assets.
+
+#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) 
+If the total Sagittarius Engine debt is less than the Soft Debt Ceiling, this is set to be equal to the [Dai Savings Rate](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate). It increases exponentially with the amount of Sagittarius Engine debt that exceeds the Soft Debt Ceiling.
```
Comment (blimpa) [inline tokenomics/sagittarius-engine.md:26]: (The soft DC remains stationary despite applications of the DC-IAM an is changed through governance action, right?)

File: tokenomics/sagittarius-engine.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+#### Price Oracle
+The oracle for MKR price uses [Elixir](elixir.md) as the primary data source. 
+
+A "sticky price" feature limits upward movement of the price within a fixed time period. That is, Maker Governance can choose to limit the speed at which the reported price of MKR increases, regardless of how fast the Elixir price is increasing. This constraint does not apply to downward price movement.
```
Comment (blimpa) [inline tokenomics/sagittarius-engine.md:31]:
```suggestion
A "sticky price" feature can be activated to limit the upward movement of the price within a fixed time period. That is, Maker Governance can choose to limit the speed at which the reported price of MKR increases, regardless of how fast the Elixir price is increasing. This constraint does not apply to downward price movement. 
```

**Review by blimpa — CHANGES_REQUESTED** [review]

File: tokenomics/sagittarius-engine.md (hunk 2 above)
Comment (astronautthis) [reply tokenomics/sagittarius-engine.md:26]: Yes, correct

**Pull request merged**